tests/modules/services/window-managers/sway/sway-null-config.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    wayland.windowManager.sway = {
+      enable = true;
+      config = null;
+      systemdIntegration = false;
+    };
+
```

Comment:
Similarly
```suggestion
    nixpkgs.overlays = [
      (self: super: {
        dmenu = dummy-package // { outPath = "@dmenu@"; };
        rxvt-unicode-unwrapped = dummy-package // {
          outPath = "@rxvt-unicode-unwrapped@";
        };
        i3status = dummy-package // { outPath = "@i3status@"; };
        sway = dummy-package // { outPath = "@sway@"; };
        xwayland = dummy-package // { outPath = "@xwayland@"; };
      })
    ];

```

Reply:
We should move these overlays to a common file for these tests.